src/main/java/fi/aalto/cs/apluscourses/intellij/services/Dialogs.java

```diff
@@ -0,0 +1,22 @@
+package fi.aalto.cs.apluscourses.intellij.services;
+
```

Comment:
We should perhaps reconsider the naming of this package, since neither `Dialogs` nor `PluginSettings` are services in the IntelliJ SDK sense